### ISSUE TITLE
Settings > Reading > Add feed and follower settings

### ISSUE THREAD

**Issue by rickybanister** [Automattic/wp-calypso#15570]
Jetpack adds support for subscriptions and a few options to customize subscriptions.

Those settings have not been replicated on WordPress.com

![image](https://user-images.githubusercontent.com/437258/27602445-ce6e8b4e-5b40-11e7-8f58-fd02dbc8dff5.png)

We need to facilitate a place for these. Most likely in 'discussion' to live in the Subscriptions setting area. 

cc @oskosk @tyxla

**Comment by tyxla** [comment]
Still valid if we decide to prioritize it.

**Comment by jasmussen** [comment]
@tyxla Is this still valid today?

**Comment by tyxla** [comment]
I don't think so, no. This was for creating settings pages in WordPress.com, but these days we're going the other way around, so this should be closed.

Nowadays, Jetpack has this in Newsletter settings:

![Image](https://github.com/user-attachments/assets/f9156687-3156-4da6-8913-ce85c96f97f3)

I'm not sure about comments (maybe we no longer send an email for that)?

In any case, this is irrelevant now, so closing.
